src/components/menu/menu.tsx

```diff
@@ -67,6 +105,37 @@ export class Menu {
     @Prop({ reflect: true })
     public gridLayout = false;
 
+    /**
+     * A search function that takes a search-string as an argument,
+     * and returns a promise that will eventually be resolved with
+     * an array of `MenuItem`:s.
+     *
+     * See the docs for the type `Searcher` for type information on
```

Comment:
Not sure if this works or not, otherwise it should just be `MenuSearcher` 
```suggestion
     * See the docs for the type {@link MenuSearcher} for type information on
```

Reply:
It works, and is fixed now.